Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -679,56 +679,73 @@ class TPCTimeSeries : public Task
       }
 
       GPUCA_DEBUG_STREAMER_CHECK(if (o2::utils::DebugStreamer::checkStream(o2::utils::StreamFlags::streamTimeSeries)) {
-        auto clusterMask = makeClusterBitMask(track);
-        const auto& trkOrig = tracksTPC[iTrk];
-        const bool isNearestVtx = (idxITSTPC.back() == -1); // is nearest vertex in case no vertex was found
-        const float mx_ITS = hasITSTPC ? tracksITSTPC[idxITSTPC.front()].getX() : -1;
-        int typeSide = 2; // A- and C-Side cluster
-        if (track.hasASideClustersOnly()) {
-          typeSide = 0;
-        } else if (track.hasCSideClustersOnly()) {
-          typeSide = 1;
+        const float sqrts = 13600; // o2::base::GRPGeomHelper::instance().getGRPLHCIF()->getSqrtS();
```

Comment:
Hello @shahor02 
for the downsampling with the tsalis we also need the centre of mass energy, which should be the 13.6TeV for pp.
I tried to get it with `o2::base::GRPGeomHelper::instance().getGRPLHCIF()->getSqrtS();` as you see, but somehow it returned `[2828107:tpc-time-series]: [21:21:45][INFO] sqrts: 113324`. Is there some other way to get the 13.6TeV or am I missing something?

Reply:
@matthias-kleiner sorry, just noticed your question. LHC DIP provides the energy in units of 0.12 GeV. I've added methods to get it in GeV, also the `getSqrtS()` method will return GeVs now: https://github.com/AliceO2Group/AliceO2/pull/12120.

Reply:
Thank you for the clarification! I will then add it later